src/main/java/couse_project/RandomIntelligence.java

```diff
@@ -0,0 +1,26 @@
+package couse_project;
+
+import java.util.Random;
+
+public class RandomIntelligence {
+    private SeaMap map;
+    private final SeaMap radar;
+
+    public RandomIntelligence(SeaMap map, SeaMap radar) {
+        this.map = map;
+        this.radar = radar;
+    }
+
+    public boolean shoot() {
+        Random random = new Random();
+        int x = random.nextInt(10);
+        int y = random.nextInt(10);
+
+        if (radar.isFreeEnterCell(x, y)) {
+            return radar.registerShot(x, y);
+        }
+
+        shoot();
+        return false;
```

Comment:
Same here: What's the case when it will return false? Seems impossible

Reply:
let me check and fix if it is needed